### DISCUSSION TITLE
Cannot remove a deleted accounts in account list (Multiple GitHub accounts on GitHub Mobile)

### DISCUSSION THREAD

**Discussion by Super12138** [community/community#81661] [Mobile]
### Select Topic Area

Bug

### Body

I try the new features multiple GitHub accounts on GitHub Mobile with my another account. After try that, I log out that account. A few days later, I deleted that accounts. But when I open GitHub Mobile again, it still show in the accounts list!
![Can't remove the accounts](https://github.com/community/community/assets/70494801/c4c29067-7be2-47ea-8cb8-41df5331c9b9)
I tried to delete it many times, and no matter how much I tap the logout button, it couldn't be deleted.

**Comment by harrylal** [comment]
Hi @Super12138 **Have you tried removing the cache?**
Go to your mobile device's settings, find the GitHub Mobile app, and clear its cache. Clearing the cache can often remove any stored data related to the deleted account.

**Reply by Super12138** [reply]
> Hi @Super12138 **Have you tried removing the cache?**
> Go to your mobile device's settings, find the GitHub Mobile app, and clear its cache. Clearing the cache can often remove any stored data related to the deleted account.

Yes, I uninstall the GitHub Mobile app, and reinstall in Play Store. It is still has this problem.

**Reply by harrylal** [reply]
Hi, @Super12138  
I see you've attempted reinstalling, but I'd recommend explicitly clearing the cache once. You might want to explore the reasons behind this suggestion[here](https://stackoverflow.com/questions/70290806/android-uninstall-app-vs-clear-all-data-not-only-cache)

**Comment by rafaelplacido86882-art** [comment]
delete mi accon all telephone external  name : rafael placido basurto /01/11/1990

**Comment by syedabdulbasitali1** [comment]
Hi! This happens because the app has cached the login token locally, but since the account is deleted on the server, the "Logout" action fails (because the server can't acknowledge the request for an account that no longer exists).

Basically, the app is stuck in a loop. Here is how to force-remove it:

Method 1: Clear App Data (Recommended)

If you are on Android:

Go to Settings > Apps > GitHub.

Select Storage & Cache.

Tap Clear Storage/Data (Clearing just the cache is usually not enough).

Open the app and log in again with your active account.

If you are on iOS:

iOS doesn't allow clearing app data easily. The fastest way is to Delete the App entirely.

Restart your phone (to clear temporary cache).

Reinstall GitHub from the App Store.

Method 2: Remove from Account Switcher (if accessible) Sometimes the "Logout" button fails, but the "Remove" option works:

Open the Profile tab.

Long-press the profile picture (or tap the gear icon).

Look for an "Edit" or "Manage Accounts" button in the list.

Try to swipe the deleted account to the left to see if a "Remove" options appears, rather than trying to switch to it and logout.

Method 1 usually fixes this 100% of the time. Hope this helps!
